Documentation-rendertest/Typesetting/Index.rst

```diff
@@ -1,170 +1,132 @@
+..  _emphasis:
 
-==============
-Typesetting H1
-==============
+Emphasis
+========
 
-Just a line.
+Emphasizing text can be done using asterisks or underscores:
 
-Typesetting H2
-==============
+*This text is emphasized.* _This text is also emphasized._
 
-Just a line.
+Code
+====
 
-Typesetting H3
---------------
+Inline code
+-----------
 
-Just a line.
+You can highlight inline code using backticks: `some code`, :php:`$somePHP`.
 
-Typesetting H4
-~~~~~~~~~~~~~~
+Code blocks
+-----------
 
-Just a line.
+For displaying larger code snippets, use code blocks:
 
-Typesetting H5
-++++++++++++++
+.. code-block:: php
 
-Just a line.
+    <?php
+    function greet($name) {
+        echo "Hello, $name!";
+    }
 
-Typesetting H6
-##############
+    greet("world");
 
-Just a line.
+Lists
+=====
 
+Lists can be unordered or ordered:
 
-Typesetting H2
-==============
+Unordered List:
 
-Typesetting is the composition of text by means of arranging physical
-types[1] or the digital equivalents. Stored letters and other symbols
-(called sorts in mechanical systems and glyphs in digital systems)
-are retrieved and ordered according to a language's orthography for
-visual display.
+*   Item 1
+*   Item 2
 
-- one
-- two Typesetting requires the prior process of designing a
-  font (which is widely but erroneously confused with and substituted for
-  typeface). One significant effect of typesetting was that authorship of
-  works could be spotted more easily; making it difficult for copiers who
-  have not gained permission.
+    *   Subitem 1
+    *   Subitem 2
 
-- three
+*   Item 3
 
-Typesetting requires the prior process of designing a
-font (which is widely but erroneously confused with and substituted for
-typeface). One significant effect of typesetting was that authorship of
-works could be spotted more easily; making it difficult for copiers who
-have not gained permission.
+Ordered List:
 
-*Typesetting requires the prior process of designing a
-font (which is widely but erroneously confused with and substituted for
-typeface). One significant effect of typesetting was that authorship of
-works could be spotted more easily; making it difficult for copiers who
-have not gained permission.*
+1.  First item
+2.  Second item
+3.  Third item
 
-**Typesetting requires the prior process of designing a
-font (which is widely but erroneously confused with and substituted for
-typeface). One significant effect of typesetting was that authorship of
-works could be spotted more easily; making it difficult for copiers who
-have not gained permission.**
 
-ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz
+Blockquotes
+===========
 
-*ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz*
+You can include blockquotes by indenting them:
 
-**ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz**
+    This is a blockquote.
+    It can span multiple lines.
 
+Tables
+======
 
-Typesetting H3
---------------
+Tables are represented using pipes and dashes:
```

Comment:
What about CSV tables?

Reply:
There are extended table examples. This should just give a quick overview for those who work on the styles

Reply:
Please add it to the introduction of the page, that it is just a quick overview.

Reply:
done, please have another look